README.md

```diff
@@ -1,6 +1,7 @@
 # ![Project Link app](https://raw.githubusercontent.com/fxbox/app/master/app/img/icons/32.png "Project Link app") Project Link app
 
 [![Build Status](https://travis-ci.org/fxbox/app.svg?branch=master)](https://travis-ci.org/fxbox/app)
+![Documentation Coverage](https://rawgit.com/fxbox/assets/badge.svg)
```

Comment:
We can't serve SVG from github directly, so I have to use rawgit.com.
We have to wait until this PR lands to be able to get the image.
So at this point there's no preview possible.